ProjectLighthouse/Helpers/CensorHelper.cs

```diff
@@ -17,7 +19,7 @@ public static class CensorHelper
         "UwU", "OwO", "uwu", "owo", "o3o", ">.>", "*pounces on you*", "*boops*", "*baps*", ":P", "x3", "O_O", "xD", ":3", ";3", "^w^",
     };
 
-    public static string FilterMessage(string message)
+    public static string FilterMessage(string message, string filterLocation = null, string username = null)
```

Comment:
```suggestion
    public static string FilterMessage(string message, FilterLocation filterLocation = null, string username = null)
```
An enum would probably be better for the filter location, you'd have to create this yourself (don't commit this directly)

Reply:
I agree, I felt like using a string was kinda janky. I'll fix this tomorrow.